CMakeLists.txt

```diff
@@ -510,6 +510,18 @@ install(TARGETS celeritas_device_toolkit
   EXPORT celeritas-targets
 )
 
+#----------------------------------------------------------------------------#
+if(APPLE)
```

Comment:
Why limit this test to apple?  We think only Apple provides it now, but other platforms may be providing now or later without us knowing.